ConsentViewController/Classes/SourcePointClient/SourcePointClient.swift

```diff
@@ -348,6 +348,61 @@ class SourcePointClient: SourcePointProtocol {
     }
 
     func customConsentGDPR(
+        toConsentUUID consentUUID: String,
+        vendors: [String],
+        categories: [String],
+        legIntCategories: [String],
+        propertyId: Int,
+        handler: @escaping AddOrDeleteCustomConsentHandler) {
+            coreClient.customConsentGDPR(
+                consentUUID: consentUUID,
+                propertyId: Int32(propertyId),
+                vendors: vendors,
+                categories: categories,
+                legIntCategories: legIntCategories) { response, error in
+                    if error != nil || response == nil {
+                        self.fallbackCustomConsentGDPR(
+                            toConsentUUID: consentUUID,
+                            vendors: vendors,
+                            categories: categories,
+                            legIntCategories: legIntCategories,
+                            propertyId: propertyId,
+                            handler: handler
+                        )
+                    } else {
+                        handler(Result.success(response!.toNativeAsAddOrDeleteCustomConsentResponse())) // swiftlint:disable:this force_unwrapping
```

Comment:
Is it safe to unwrap here?

Reply:
It should be. We check the response for being null in the `if` statement above